SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -96,7 +96,7 @@ void CIncNSSolver::Preprocessing(CGeometry *geometry, CSolver **solver_container
     SetPrimitive_Limiter(geometry, config);
   }
 
-  ComputeVorticityAndStrainMag<1>(*config, iMesh);
```

Comment:
What did the <1> here signify?

Reply:
It was part of a way to make a general function for all solvers.  It is essentially VEL_INDEX